mkt/ratings/tests/test_models.py

```diff
@@ -38,3 +38,15 @@ def test_review_aggregates_triggers_reindex(self, index_webapps):
         index_webapps.reset_mock()
         addon_review_aggregates(self.app.pk)
         assert index_webapps.called
+
+    def test_soft_delete(self):
+        eq_(Review.objects.count(), 2)
+        eq_(Review.with_deleted.count(), 2)
+
+        Review.objects.all()[0].delete()
+        eq_(Review.objects.count(), 1)
+        eq_(Review.with_deleted.count(), 2)
+
+        Review.objects.all()[0].delete()
+        eq_(Review.objects.count(), 0)
+        eq_(Review.with_deleted.count(), 2)
```

Comment:
I'm not sure the second set of assertions show you much here either. Unless you are expecting something wierd on the second deletion.